File: kafka-client-ex/src/main/java/in/srini91/learn/kafkaclientex/dao/model/Person.java
package in.srini91.learn.kafkaclientex.dao.model;

import lombok.Data;

import javax.persistence.*;

@Entity
@Table(name = "person")
@Data
public class Person {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;

    private String name;

    private Integer age;

    private String address;

}

File: kafka-cloud-stream-ex/src/main/kotlin/srini/learn/kafka/kafkacloudstreamex/dao/model/Person.kt
package srini.learn.kafka.kafkacloudstreamex.dao.model

import lombok.Data
import javax.persistence.*

@Entity
@Table(name = "person")
class Person(
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
     var id: Long? = null,

     var name: String?=null,

     var age: Int? = null,

     var address: String? = null


)